chat-plugins/mafia.js

```diff
@@ -151,7 +151,7 @@ function parseRole(roleString) {
 		memo: ['During the Day, you may vote for whomever you want lynched.'],
 		alignment: '',
 	};
-	roleString = roleString.replace(/\s*\(.*?\)\s*/g, ' ');
+	roleString = roleString.replace(/\s*\(.*\)/g, ' ');
```

Comment:
Seems like a mostly good change, but there are unintended side affects.

```
'hello (remove me) and me) world'.replace(/\s*\(.*\)/g, ' ');
"hello  world"

'hello (remove me (and me) world'.replace(/\s*\(.*?\)\s*/g, ' ');
"hello world"

'hello (remove (all of) this) world'.replace(/\s*\(.*\)/g, ' ');
"hello  world"

'(just remove this) hello world (and this)'.replace(/\s*\(.*\)/g, ' ');
" "
```
In test case's 1 & 3, theres 2 spaces between `hello` and `world`, resulting in a `""` being passed to the parser.
In test case 4, `["", ""]` is passed to the parser (it wiped the whole role out).

Reply:
oh, 1/3 is because i didnt realise you were replacing with ' ', my bad

honestly ill just revert this, i dont think it's too much an issue to require hosts to match their backets